spec/controllers/api/v1/providers_controller_spec.rb

```diff
@@ -20,5 +20,16 @@
 
       get :index
     end
+
+    it 'renders a 400 when the changed_since param is not valid' do
+      allow(controller).to receive(:authenticate)
+
+      get :index, params: { changed_since: '2019' }
+      expect(response).to have_http_status(:bad_request)
+      json = JSON.parse(response.body)
+      expect(json). to eq(
+        'status' => 400, 'message' => 'Invalid changed_since value, the format should be a iso8601 timestamp'
```

Comment:
I don't think having the exact message letter for letter in the test is important, it couples the implementation unnecessarily, but it's a very minor thing